docs/_data/nav.yml

```diff
@@ -6,9 +6,9 @@
   href: /support/
   category: support
 
-- title: Blog
-  href: /blog/
-  category: blog
+# - title: Blog
+#   href: /blog/
+#   category: blog
```

Comment:
I think it might be best to remove this completely, along with updating the blog directory itself so that it can't be accidentally stumbled upon by people who know the link. In fact, maybe we redirect the blog page to the Nuclide home page or something.

So, change https://raw.githubusercontent.com/facebook/nuclide/master/docs/blog/index.md and remove all the Jekyll `{% .. %}` stuff and put a redirect under `title: Blog`  that says:

```
layout: redirect
destination:  /
```

Something like that.